ecommerce/colaraz_features/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url, include
+
+from ecommerce.colaraz_features import views
+from ecommerce.colaraz_features.api import urls
+
+app_name = 'colaraz_features'
+
+urlpatterns = [
+    url(r'^api/v1/', include(urls.urlpatterns)),
```

Comment:
Do we need to add a namespace here?

Reply:
We don't need one here as the `colaraz_features` namespace will be used here as well, however adding one now.